content/competitions/codejam.html

```diff
@@ -8,19 +8,15 @@
     <section class="column is-6">
 
         <div class="block">
-            <a href="https://www.facebook.com/events/659245534649313/">
-                <img src="{{< baseurl >}}/img/media/codejam_banner.png" style="width: 100%" />
+            <a href="https://fb.me/e/42v91HFQA">
+                <img src="/img/media/codejam_banner.png" style="width: 100%" />
             </a>
         </div>
 
-        <h1 class="title is-3">UQCS CodeJam 2020 (presented by Atlassian)</h1>
-        <h2 class="subtitle is-4">10am &ndash; 3:30pm, Saturday 28th March</h2>
+        <h1 class="title is-3">UQCS CodeJam 2021 (Presented by IMC)</h1>
```

Comment:
Why does the art say "supported by" and this say presented by?

Reply:
it gives ✨ c h a r a c t e r ✨